docs/docs/using-mdx.mdx

```diff
@@ -361,27 +362,27 @@ console.log(
       }
     }}
   />
-)
+}
 ```
 
 …we’d get:
 
-{
-  <div className="card">
-    <ul>
-      <li><a href="#alpha" style={{borderTop: '1px dotted', color: 'violet'}}>markdown syntax</a></li>
-      <li><a href="#bravo">JSX with a lowercase name</a></li>
-      <li><a href="#charlie" style={{borderTop: '1px dashed', color: 'tomato'}}>JSX with a capitalized name</a></li>
-    </ul>
-  </div>
-}
+```jsx
+<ul>
+  <li><a href="#alpha" style={{borderTop: '1px dotted', color: 'violet'}}>markdown syntax</a></li>
+  <li><a href="#bravo">JSX with a lowercase name</a></li>
+  <li><a href="#charlie" style={{borderTop: '1px dashed', color: 'tomato'}}>JSX with a capitalized name</a></li>
+</ul>
+```
+
+Observe that:
 
-Observe that the first link (`#alpha`) is dotted and violet.
-That’s because `a` is the HTML equivalent for the markdown syntax being used.
-The second link (`#bravo`) remains unchanged,
-because in JSX syntax `a` is a literal tag name.
-The third link (`#charlie`) is dashed and tomato,
-as in JSX syntax `Link` is a reference.
+1. The first link (`#alpha`) is dotted and violet, because `a` is the HTML
+   equivalent for the markdown syntax `[]()`.
+2. The second link (`#bravo`) remains unchanged, because in JSX syntax `a` is a
+   literal tag name, which [MDX does not modify][mdx-no-literals-modification].
```

Comment:
Explained more in https://github.com/mdx-js/mdx/issues/2503#issuecomment-2218365581
There's more value in describing what MDX _is_, I'd like to avoid the rabbit whole of trying to list all the things MDX isn't.

> The second link (`#bravo`) is rendered as-is, because in JSX syntax `a` is a
>   literal tag name.

Reply:
My reasoning behind this and behind #2503 is that:

- I think it could be a misconception that should be broken
- on a docs page about MDX, I wouldn't find it unuseful or off-topic to connect a note about "what JSX does" with context about how that connects to MDX (instead of leaving it to the user to fill in the blanks implicitly by reading other docs) 

Just my opinion and suggestion though, if not accepted then I'll revert this

Reply:
> I think it could be a misconception that should be broken

It is assumed knowledge from JSX. Then it is covered in-depth in “What is MDX”. Then it is explained in excruciating detail in the paragraphs above. **Then** this example covers it. No more words are needed.

You wrote a long issue about how you went down a rabbit hole by reading old issues. I do not want to send users down rabbit holes of 5 year old issues.